### PR TITLE
feat: new compontent Skeleton

### LINKED ISSUE #774: New Component - ElSkeleton
- As a developer, I want a feature that I can have a skeleton component for indicating loading state.
   - As a developer I want the skeleton to have background color consistent with [Color](https://element-plus.org/#/zh-CN/component/color) Placeholder Text with color value `#C0C4CC`
   - As a developer I want the skeleton to be animated with flag `moving`
   - As a developer I want the skeleton to have the ability to indicate `loading` state via a `bool` type flag.
   - As a developer I want the skeleton to be able to render child components when the `loading` flag is false.

### PR DESCRIPTION
**Pull request by porcelainHeart**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/element-plus/index.ts [hunk 1]
```diff
@@ -64,6 +64,7 @@ import ElRate from '@element-plus/rate'
 import ElRow from '@element-plus/row'
 import ElScrollbar from '@element-plus/scrollbar'
 import ElSelect from '@element-plus/select'
+import ELSkeleton from '@element-plus/skeleton'
```
Comment (zazzaz) [inline packages/element-plus/index.ts:67]: El

**Comment by zazzaz** [comment]
[picture-outline.svg.zip](https://github.com/element-plus/element-plus/files/5721988/picture-outline.svg.zip) 

@JeremyWuuuuu SVG file of picture-outline icon

**Review by zazzaz — APPROVED** [review]

File: website/nav.config.json [hunk 2]
```diff
@@ -1404,6 +1420,10 @@
             {
               "path": "/badge",
               "title": "Badge"
+            },
+            {
+              "path": "/skeleton",
+              "title": "Skeleton 骨架屏"
```
[the remark this answers is not in the document]
Comment (JeremyWuuuuu) [reply website/nav.config.json]: Done.

File: website/nav.config.json [hunk 3]
```diff
@@ -1098,6 +1110,10 @@
...
```
[the remark this answers is not in the document]
Comment (JeremyWuuuuu) [reply website/nav.config.json]: Done

**Pull request merged**